query_designer/views.py

```diff
@@ -61,7 +61,7 @@ def simplified(request, pk=None):
 
 
 def clean(request, pk=None):
-    storage_target = 'LOCAL_POSTGRES'
+    storage_target = 'UBITECH_POSTGRES'
```

Comment:
This is a change needed in order to show all the datasets instead of the local ones